features/user/account/renewing_account_after_deletion.feature

```diff
@@ -0,0 +1,18 @@
+@account
+  Feature: Registering account again after it has been deleted
```

Comment:
Actually I would remove this feature from this PR and do it in a separate one. For now let's implement only `Deleting the account in backend`